data/architectures.json

```diff
@@ -146,6 +146,13 @@
             "onnx"
         ]
     },
+    "realplksr-dysample": {
+        "name": "RealPLKSR_dysample",
+        "input": "image",
+        "compatiblePlatforms": [
+            "pytorch"
+        ]
+    },
```

Comment:
Why did you make this a new arch? Isn't dynsample just a hyperparameter for RealPLKSR?

Or does ONNX not support dynsample? In that case, I think a new arch would be fine.

Reply:
You're right, it is a hyperparameter. I was thinking of it in terms of compatibility, my bad.

To my knowledge it does support ONNX, but i can't be 100% sure about that. I'm asking someone on EE right now

Reply:
It supports statix onnx only. In this case, would you like me to remove the additional arch? @RunDevelopment